templates/resources/xl-deploy/default-conf/xl-deploy.conf.template.j2

```diff
@@ -1,18 +1,9 @@
 xl {
-  # The cluster setup should be in system.conf
-  #cluster {
-  #  # mode: "default", "hot-standby", "full"
-  #  mode = ${XL_CLUSTER_MODE}
-  #  name = "{{ product }}_cluster"
-  #  akka {
-  #    loglevel = "INFO"
-  #    actor.debug.receive = off
-  #    remote {
-  #      log-received-messages = off
-  #      log-sent-messages = off
-  #    }
-  #  }
-  #}
+
+  server {
+    hostname = ${XL_NODE_NAME}
```

Comment:
This one should also be in the `node-conf` directory. Between restarts of a container the IP address will change. The configuration file in `node-conf` will always be regenerated on start and cannot be overridden by the user. The one in default-conf is used to bootstrap an empty container, but not for a volume mapped conf directory.

Reply:
@hierynomus It IS also in node-conf.
Are you saying it should NOT be here?

Reply:
@hierynomus ah, I see, the property is not there, just the cluster configuration.